ESPController/src/main.cpp

```diff
@@ -1439,11 +1439,11 @@ static void startMDNS()
 static void event_handler(void *arg, esp_event_base_t event_base,
                           int32_t event_id, void *event_data)
 {
-
-  // ADD IN...
-  // IP_EVENT_STA_LOST_IP
-
-  if (event_base == WIFI_EVENT && event_id == WIFI_EVENT_STA_START)
+  if (event_base == WIFI_EVENT && event_id == WIFI_EVENT_STA_BSS_RSSI_LOW)
+  {
+    ESP_LOGW(TAG, "WiFi signal strength low");
```

Comment:
Can you add RSSI to the TFT as well? Perhaps for this event also have the TFT display a warning screen?

Reply:
Yep, added it back in just now.